src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -193,14 +193,14 @@ private void AnalyzeDirectory(DirectoryInfo directoryPath, ReportFormat reportFo
                 if (!directoryPath.Exists)
                 {
                     logger.LogError("Invalid directory: {directoryPath}", directoryPath);
-                    return;
+                    return 2;
                 }
 
                 // Check that output file path provided for sarif report
                 if (reportFormat == ReportFormat.Sarif && outputFilePath == null)
                 {
                     logger.LogError("Output file path is not provided.");
```

Comment:
```suggestion
                    logger.LogError("Output file path was not provided.");
```

Unrelated to the PR, but I just realized we are outputting two different messages for the same error

Reply:
I don't see it. Where is the 2nd instance of this error?

Reply:
Lines 144 and 202

Reply:
@VeraBE can you explain?  What are the two messages, for what error?

Reply:
144 is only logged if `AnalyzeTemplate()` is not called from `AnalyzeDirectory()`, where `AnalyeDirectory()` will log it instead.  This is another area that will be improved with #207.